frontend/src/metabase/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -1,9 +1,12 @@
 import cx from "classnames";
 import type { ReactNode } from "react";
 
+import CS from "metabase/css/core/index.css";
+import QueryBuilderS from "metabase/css/query_builder.module.css";
+
 interface ErrorMessageProps {
   title: string;
-  type: string;
+  type: "timeout" | "serverError" | "noRows";
```

Comment:
how could you determine these values? I'm a bit confused because `QueryErrorImageTimeout` didn't originally exist. And there's only another reference to `.QueryError-image--serverError` which I think could be a type of `serverError`.

In short, why do we have `timeout` instead of `serverError` here?

Reply:
Gotcha